compiler-core/src/language_server/completer.rs

```diff
@@ -447,7 +447,7 @@ where
                     insert_range,
                     TypeCompletionForm::Default,
                 );
-                add_import_to_completion(&mut completion, import_location, module_full_name);
+                add_import_to_completion(&mut completion, import_location, module_full_name, false);
```

Comment:
How come this is always false?

Reply:
Good catch, I assumed it's import. Fixed it.